.github/workflows/stage.yaml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - dev
```

Comment:
Please, disable it for the `dev` branch. Read [more details in the Slack](https://cere-network.slack.com/archives/C01ALH83YVB/p1713956080572079?thread_ts=1713876820.091239&cid=C01ALH83YVB)